Makefile

```diff
@@ -56,6 +59,17 @@ mnesia_backup:
 		-e backup_suffix=$(BACKUP_SUFFIX) \
 		mnesia_backup.yml
 
+~/.ssh/id_ae_infra_ed25519:
+	@ssh-keygen -t ed25519 -N "" -f $@
+
+~/.ssh/id_ae_infra_ed25519-%-cert.pub: ~/.ssh/id_ae_infra_ed25519
+	@vault write -field=signed_key ssh/sign/$* public_key=@$<.pub > $@
+
+ssh-%: ~/.ssh/id_ae_infra_ed25519-%-cert.pub
+	@ssh -i $< -i ~/.ssh/id_ae_infra_ed25519 $*@$(HOST)
+
+ssh: ssh-epoch
```

Comment:
I read intended usage as `make ssh HOST=...` after running image attaching to terminal (`-it`). Is user allowed to do make ssh... to a host then exit from that host then run make ssh... for other host? I expect yes. In this case:
* Is there any assumption that user does not present distinct pubkeys/certs in a short succession?
* Is this makefile meant to keep privkey, pubkey and cert between 2 invocations of `make ssh...`? If so, I did not check whether intermediate targets `~/.ssh/id_ae_infra_ed25519-%-cert.pub` and `~/.ssh/id_ae_infra_ed25519` are removed at end of first invocation (while pubkey `*.pub` is for sure kept).

Reply:
> * Is there any assumption that user does not present distinct pubkeys/certs in a short succession?

I'm sorry, I don't get this one, can you rephrase please?

Reply:
> * Is this makefile meant to keep privkey, pubkey and cert between 2 invocations of `make ssh...`? If so, I did not check whether intermediate targets `~/.ssh/id_ae_infra_ed25519-%-cert.pub` and `~/.ssh/id_ae_infra_ed25519` are removed at end of first invocation (while pubkey `*.pub` is for sure kept).

The cert is removed, however it's more a side effect. The makefile keeps the keys to slightly reduce the connecting time. The certificate itself have expiration time, the user is free to reuse it until it expires.

Reply:
> > * Is there any assumption that user does not present distinct pubkeys/certs in a short succession?
> 
> I'm sorry, I don't get this one, can you rephrase please?

* User runs `make ssh...` for accessing a host.
* User exits from that host (but stays in Docker container).
* User runs `make ssh...` for accessing a host (same as before, or distinct).

I understand from your other answer ("The cert is removed ... The makefile keeps the keys ...") that the cert is removed before the second `make ssh...` so a new cert is generated. My question was about any throttling/restriction that would prevent accessing multiple hosts, e.g. running script in container accessing 20 hosts one after another would attempt to generate 20 certs.

* Case A: If number of generated certs throttled/restricted, multiple `make ssh...` would fail.
* Case B: The accessed host (not the Docker container, but the host to which user SSHs into) grants access to user on the basis on multiple distinct certs. I assume there is no throttling/limit to how many distinct certs the host grants access to.

Reply:
I'm not aware of any  throttling/limits so far.